pytest.ini.example

```diff
@@ -23,8 +23,6 @@ env =
     azext_pnp_repository=
     azext_pnp_cs=
     azext_iot_central_app_id=
-    azext_iot_central_device_id=
```

Comment:
Great, happy to see tests being easier to run. Can you please also remove the the relevant env vars from the [set-testenv-sentinal](https://github.com/Azure/azure-iot-cli-extension/blob/dev/.azure-devops/templates/set-testenv-sentinel.yml#L19) CI template?